OracleOGG/MySQL/12.2.0.1.1/Dockerfile

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#
+# Since: June, 2017
+# Author: hadi.koesnodihardjo@oracle.com
+# Description: script to build a Docker image for Oracle GoldenGate for non-Oracle (MySQL).
+#
+#
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+#
+# Copyright (c) 2016-2017 Oracle and/or its affiliates. All rights reserved.
+#
+#
+FROM mysql/mysql-server:5.7.18
+MAINTAINER OGG QA <hadi.koesnodihardjo@oracle.com>
+#Container Additional Environment Variables
+ENV OGG_SHIPHOME=V100691-01.zip \
+    OGG_HOME=/u01/app/oracle/product/12.2.0.1.1/oggmysql_1 \
+    OGG_PORT=1700 \
+    STAGE_SOFTWARE=/u01/software \
+    MYSQL_ROOT_PASSWORD=pakalolo \
```

Comment:
Do not hard-code a password. This must be automatically generated on container start (not build) or provided by the end-user using `-e`.